index.html

```diff
@@ -55,5 +55,147 @@ <h1 class="hello-h1">Hello, I am a Frontend Developer</h1>
                 <a href="#"><i class="fa-brands fa-github"></i></a>
             </div>
         </section>
+        <main>
```

Comment:
Hi estimed @sazukeR, you have done a great Job! 💯 
I suggest you use tag <ul> to make the bar of social networks.
Because the main objetive is can use the TAB key for page's accesibility.